spec/operators/bufferToggle-spec.ts

```diff
@@ -1,352 +1,414 @@
-
-    let i = 0;
-    const result = e1.pipe(bufferToggle(e2, () => closings[i++]));
-
-    expectObservable(result).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(subs);
-    expectSubscriptions(closings[0].subscriptions).toBe(closeSubs[0]);
-    expectSubscriptions(closings[1].subscriptions).toBe(closeSubs[1]);
+    testScheduler.run(({ hot, cold, expectObservable, expectSubscriptions }) => {
+      const e1 = hot('--a--^---b---c---d---e---f---g---h------|');
+      const e2 = cold('    --x-----------y--------z---|        ');
+      const subs = '       ^-------------!                     ';
+      const closings = [
+        cold('             ---------------s--|                 '),
+        cold('                         #                       ')
+      ];
+      const closeSubs =  [
+        '                  --^-----------!                     ',
+        '                --------------(^!)                    '
+      ];
+      const expected = '   --------------#                     ';
+
+      let i = 0;
+      const result = e1.pipe(bufferToggle(e2, () => closings[i++]));
+
+      expectObservable(result).toBe(expected);
+      expectSubscriptions(e1.subscriptions).toBe(subs);
+      expectSubscriptions(closings[0].subscriptions).toBe(closeSubs[0]);
+      expectSubscriptions(closings[1].subscriptions).toBe(closeSubs[1]);
+    });
   });
 
   it('should propagate error emitted late from a closing', () => {
-    const e1 = hot('--a--^---b---c---d---e---f---g---h------|');
-    const e2 =     cold('--x-----------y--------z---|        ');
-    const subs =        '^                  !                ';
-    const closings = [
-      cold(             '---------------s--|               '),
-      cold(                         '-----#                ')];
-    const closeSubs =  ['  ^              !                  ',
-                      '              ^    !                '];
-    const expected =    '-----------------i-#                ';
-    const values = {
-      i: ['b', 'c', 'd', 'e']
-    };
-
-    let i = 0;
-    const result = e1.pipe(bufferToggle(e2, () => closings[i++]));
-
-    expectObservable(result).toBe(expected, values);
-    expectSubscriptions(e1.subscriptions).toBe(subs);
-    expectSubscriptions(closings[0].subscriptions).toBe(closeSubs[0]);
-    expectSubscriptions(closings[1].subscriptions).toBe(closeSubs[1]);
+    testScheduler.run(({ hot, cold, expectObservable, expectSubscriptions }) => {
+      const e1 = hot('--a--^---b---c---d---e---f---g---h------|');
+      const e2 = cold('    --x-----------y--------z---|        ');
+      const subs = '       ^------------------!                ';
+      const closings = [
+        cold('             ---------------s--|                 '),
+        cold('                         -----#                  ')];
+      const closeSubs = [ '--^--------------!                  ',
+        '                --------------^----!                  '
```

Comment:
I think this needs to be left-aligned with the `^` in `e1`. It might be better to always put the array elements on their own lines. Prettier probably will.